.env

```diff
@@ -0,0 +1,16 @@
+# In all environments, the following files are loaded if they exist,
+# the latter taking precedence over the former:
+#
+#  * .env                contains default values for the environment variables needed by the app
+#  * .env.local          uncommitted file with local overrides
+#  * .env.$APP_ENV       committed environment-specific defaults
+#  * .env.$APP_ENV.local uncommitted environment-specific overrides
+#
+# Real environment variables win over .env files.
+#
+# DO NOT DEFINE PRODUCTION SECRETS IN THIS FILE NOR IN ANY OTHER COMMITTED FILES.
+#
+# Run "composer dump-env prod" to compile .env files for production use (requires symfony/flex >=1.2).
+# https://symfony.com/doc/current/best_practices.html#use-environment-variables-for-infrastructure-configuration
+APP_ENV=prod
```

Comment:
Do we really want prod to be the default? How is this used?

Reply:
It's a little confusing, I know. Basically it doesn't matter what we call the environment, but we have to call it _something_, and Symfony defaults to "prod". Symfony does nothing with it internally. Unless we go out of our way to e.g. change configuration based on environment, it simply doesn't matter.

The second parameter, debug, _does_ matter. That should be `false` when we ship, but `true` when debugging locally.

https://symfony.com/doc/4.1/configuration/environments.html